test/integration/ci-operator-config-mirror.sh

```diff
@@ -10,7 +10,7 @@ expected="${suite_dir}/output"
 os::test::junit::declare_suite_start "integration/ci-operator-config-mirror"
 # This test validates the ci-operator-config-mirror tool
 
-os::cmd::expect_success "ci-operator-config-mirror --to-org super-priv --config-path ${actual} --whitelist-file ${suite_dir}/whitelist.yaml"
+os::cmd::expect_success "ci-operator-config-mirror --to-org super-priv --config-path ${actual} --clean=false --whitelist-file ${suite_dir}/whitelist.yaml"
```

Comment:
I already wrote this in an earlier review, but I think we should have a test that runs `--clean=true` as that will be the default, desired mode.

Reply:
I will address this. Thanks, it seems that I missed that one.